pkg/ring/kv/dynamodb/dynamodb.go

```diff
@@ -57,13 +57,12 @@ func newDynamodbKV(cfg Config, logger log.Logger) (dynamodbKV, error) {
 		return dynamodbKV{}, err
 	}
 
+	awsCfg := aws.NewConfig()
 	if len(cfg.Region) > 0 {
```

Comment:
Region here is optional in the config.   
I would need to do a test to see what the client will do if we send an empty string. I am not sure if it will try to use the env var.

Reply:
Cool just a nit